Source/mpq/mpq_writer.hpp

```diff
@@ -14,14 +14,10 @@
 namespace devilution {
 class MpqWriter {
 public:
-	bool Open(const char *path);
-
-	bool Close(bool clearTables = true);
-
-	~MpqWriter()
-	{
-		Close();
-	}
+	MpqWriter(const char *path);
+	MpqWriter(MpqWriter &&other) = default;
+	MpqWriter &operator=(MpqWriter &&other) = default;

```

Comment:
Now that you've got `unique_ptr` members, which are move-only, the class itself becomes move-only by default. These two lines become superfluous.

Reply:
The magic of the c++ compiler. 😁 
Thanks removed 🙂

Reply:
And old gcc says no...
From amigaos and linux_x86_65 build:
> /root/repo/Source/pfile.cpp: In function 'devilution::MpqWriter devilution::{anonymous}::GetSaveWriter(uint32_t)':
/root/repo/Source/pfile.cpp:164:47: error: use of deleted function 'devilution::MpqWriter::MpqWriter(const devilution::MpqWriter&)'
  return MpqWriter(GetSavePath(saveNum).c_str());

That's why I added both lines again.